services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -226,6 +251,18 @@ class HankeServiceITests : DatabaseTest() {
         assertFeaturePropertiesIsReset(result, mutableMapOf("hankeTunnus" to hanke.hankeTunnus))
     }
 
+    @Test
+    fun `getHankeHakemuksetPair maps hanke and hakemukset to a pair correctly`() {
```

Comment:
There could be tests for the cases where the hanke has no applications and where the hanke doesn't exist.

Reply:
Added two tests to verify the two mentioned scenarios.